src/compiler/marks.ts

```diff
@@ -31,7 +31,7 @@ export function compileMarks(model: Model): any[] {
       type: MARKTYPES_MAP[mark],
       from: {
         // from.data might be added later for non-facet, single group line/area
-        transform: [{type: 'sort', by: sortBy}]
+        transform: [{ type: 'sort', by: sortBy }]
```

Comment:
(This one is more annoying than the other one) -- no need to revert the change you already made -- it looks nicer but I just don't want to enforce it.  :)